cattle/utils.py

```diff
@@ -100,9 +100,10 @@ def timestamp_diff(self, time_current, time_prev):
         return round((diff * 10**9))
 
     def _timestamp_convert(self, stime):
-        # This is a bit hacky
-        t_strip = stime[:-4]
-        return datetime.strptime(t_strip, "%Y-%m-%dT%H:%M:%S.%f")
+        # Cadvisor handles everything in nanoseconds.
+        # Python does not.
+        t_conv = arrow.get(stime[0:26])
```

Comment:
Yeah, Python can't handle nanoseconds. Arrow is much more graceful dealing with the strings.